packages/server-wallet/src/wallet/store.ts

```diff
@@ -260,6 +246,69 @@ export class Store {
     )?.protocolState;
   }
 
+  async validateTransition(
+    fromState: SignedState,
+    toState: SignedState,
+    tx: Transaction
+  ): Promise<boolean> {
+    const fromMoverIndex = fromState.turnNum % fromState.participants.length;
+    const fromMover = fromState.participants[fromMoverIndex].signingAddress;
+
+    const toMoverIndex = toState.turnNum % toState.participants.length;
+    const toMover = toState.participants[toMoverIndex].signingAddress;
+    // This is the basic validation from the force move contract
+    // that applies to all state transitions
+    const basicValidation =
+      toState.turnNum === fromState.turnNum + 1 &&
+      toState.chainId === fromState.chainId &&
+      toState.participants === fromState.participants &&
+      toState.appDefinition === fromState.appDefinition &&
+      toState.challengeDuration === fromState.challengeDuration &&
+      fromState.signatures.some(s => s.signer === fromMover) &&
+      toState.signatures.some(s => s.signer === toMover);
+
+    if (!basicValidation) {
+      logger.error('Basic ForceMove transition validation failed.', {fromState, toState});
```

Comment:
💡 It's nice to have stack traces in a case like this. One way of achieving that would be 
```suggestion
      const VALIDATION_ERROR = 'Basic ForceMove transition validation failed.'
      logger.error(VALIDATION_ERROR, {fromState, toState, error: Error(VALIDATION_ERROR)});
```

(This suggestion is might not be `prettier` enough.)